src/graphnet/data/extractors/i3featureextractor.py

```diff
@@ -61,6 +62,7 @@ def __call__(self, frame) -> dict:
             x = self._gcd_dict[om_key].position.x
             y = self._gcd_dict[om_key].position.y
             z = self._gcd_dict[om_key].position.z
+            event_time = frame["I3EventHeader"].start_time.mod_julian_day_double
```

Comment:
I would add this above the `for om_key ...` loop, as the value is the same for all OMs in the event.

Reply:
Great suggestion. I have changed it.